pegs_common_utils.R

```diff
@@ -262,63 +262,374 @@ prepare_ukb_phenotype <- function(ukb_data, phenotype) {
 # Returns:-
 # A dataframe (epr_number = participant ID, Y = phenotype). All participants to
 # be excluded have their phenotype marked as NA.
-prepare_pegs_phenotype <- function(pegs_data, phenotype) {
```

Comment:
I see inconsistencies in coding style - newlines, spacing, indentation. This is easily fixed by running styler() - you can use the tidyverse style guide.